app/controllers/contributions_controller.rb

```diff
@@ -4,7 +4,7 @@ class ContributionsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @user_contributions = scopped_contributions.approved.order(pr_state: :desc)
```

Comment:
Should it be ordered only by `created_at` or also by `pr_state`?

Reply:
hmmm, good catch, I think that both, maybe the priority can be the pr_state and then the created_at. What do you think @adbatista ?

Reply:
@danielmbrasil let's leave with the created_at only.